packages/client/screens/WalletOverviewScreen.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { HomeStackScreenProps, SettingsStackScreenProps } from "../types";
+import { Pressable, Box, Text, HStack, VStack, Center } from "native-base";
+import { StyleSheet } from "react-native";
+import { faWalletCustom } from "../components/icons/faWalletCustom";
+import { FontAwesomeIcon } from "@fortawesome/react-native-fontawesome";
+import { View } from "../components/Themed";
+import { faEthereumCustom } from "../components/icons/faEthereumCustom";
+import { CompositeScreenProps } from "@react-navigation/native";
+
+type Props = CompositeScreenProps<
+    HomeStackScreenProps<"WalletOverviewScreen">,
+    SettingsStackScreenProps<"WalletOverviewScreen">
+>;
+
+export default function WalletOverviewScreen({ route, navigation }: Props) {
+    const { address, name, currencyType, balance } = route.params;
+
+    function formatWalletAddress(address: string): string {
+        return `${address.substring(0, 6)}...${address.substring(address.length - 4, address.length)}`;
+    }
+
+    return (
+        <View style={styles.view}>
+            <Box style={styles.walletDetailsWrapper}>
+                <VStack style={styles.walletDetails}>
+                    <HStack justifyContent="space-between">
+                        <VStack>
+                            <Text style={styles.walletName}>{name}</Text>
+                            <Box marginTop="2px"></Box>
+                            <Text style={styles.walletAddress}>{formatWalletAddress(address)}</Text>
+                        </VStack>
```

Comment:
Directly add the space to the VStack instead of using a Box (VStack implements Box)
```suggestion
                        <VStack space="2">
                            <Text style={styles.walletName}>{name}</Text>
                            <Text style={styles.walletAddress}>{formatWalletAddress(address)}</Text>
                        </VStack>
```